go/Internal/database/migrations/migrate.cql

```diff
@@ -0,0 +1,18 @@
+CREATE KEYSPACE IF NOT EXISTS media_player WITH replication = {'class': 'NetworkTopologyStrategy', 'replication_factor': '3'}  AND durable_writes = true;
```

Comment:
I would create a `cmd/migrate/main.go` to handle the migrations, but both... up and down. It should not be that complicated.

Reply:
We talk, and it is not necessary for this case. But thank you!